modules/@apostrophecms/ui/ui/apos/components/AposMinMaxCount.vue

```diff
@@ -39,12 +39,12 @@ export default {
   },
   computed: {
     maxed() {
-      return (this.field.max !== undefined) && (this.modelValue.length >= this.field.max);
+      return (this.field.max !== undefined) && (this.modelValue?.length >= this.field.max);
```

Comment:
I'm not sure why this protection, `modelValue` is required and has to be an array..
```javascript
    modelValue: {
      required: true,
      type: Array
    }
```

Reply:
OK, I backed it out for this file.